packages/palette/src/elements/BarChart/Bar.tsx

```diff
@@ -185,6 +188,16 @@ export const Bar = ({
         </HighlightLabel>
       )}
       {hover && label && <BarHoverLabel>{label}</BarHoverLabel>}
+      <AxisLabelX color="black60" size="2">
+        {axisLabelX}
+      </AxisLabelX>
     </BarBox>
   )
 }
+
+const AxisLabelX = styled(Sans)`
+  position: absolute;
+  top: calc(100% + ${space(0.5)}px);
+  width: 100%;
+  text-align: center;
```

Comment:
I think to be more flexible it would make sense to center this whole box instead of just the text. That would mean the box is not constrained to be the same width as the bar, and then the text will still be centered even if the text is wider than the bar.

That is, unless you want to use word wrapping in these labels? Then you'd need an explicit width like how it's done now.

Reply:
Yes good catch looking at the designs we don't want the labels to be constrained by the bar width. cc: @owendodd and @samchieng:

<img width="647" alt="Screen Shot 2019-04-25 at 10 31 03 AM" src="https://user-images.githubusercontent.com/687513/56743865-64058b00-6745-11e9-8982-7db74818d1bd.png">

Reply:
pfffff, I thought it was simple but haven't figured it out yet. Is it possible to center align the label without removing `position: absolute;`? If not it requires a good amount of change to the way bars are positioned I guess.

Reply:
try
```
left: 50%;
transform: translateX(-50%);
```

Reply:
I got that with https://github.com/artsy/palette/pull/392/commits/0ff165567c44cc0a9f0c34e4561e96b3d9e34890#diff-2e24d9a0ed8b8746d5fa5d1333da123dR231:

<img width="825" alt="Screen Shot 2019-04-25 at 12 05 16 PM" src="https://user-images.githubusercontent.com/687513/56751775-d5006f00-6754-11e9-813a-d22cb71237ef.png">

The only tricky thing was that I had to add https://github.com/artsy/palette/pull/392/commits/0ff165567c44cc0a9f0c34e4561e96b3d9e34890#diff-710f36809acea9f66cf6ae101cf56b66R13 to remove the extra padding on the right side of the chart created but `width: 200%;`

Maybe I was missing something but by some reason 👆 didn't quite work:

<img width="828" alt="Screen Shot 2019-04-25 at 12 23 00 PM" src="https://user-images.githubusercontent.com/687513/56751823-ec3f5c80-6754-11e9-9791-c4825467c3e4.png">

Reply:
Hmm I tried the left + transform trick and it looked fine.

Also while playing around I noticed that these labels aren't visible for me because of overflow issues. They don't add to the height of the bar chart, but they probably should.

It might be best to rethink how we're doing this. I'm imagining a parallel row of equal-sized flex containers underneath the row of bars which center their children using flexbox and align items using `baseline`, does that make sense to you?

```
Flex row ----->

+-------------+
|         +-+ |
| +-+     | | |
| | |     | | |
| | | +-+ | | |
| | | | | | | |
| | | | | | | |
| | | | | | | |
| +-+ +-+ +-+ |
+-------------+
+-------------+
| +-+ +-+ +-+ |
| +-+ +-+ +-+ |
+-------------+

Flex row ----->

```

Reply:
actually just tried that out and don't think it'd work because flexbox doesn't center overflowing content 😞